src/styles/style.css

```diff
@@ -0,0 +1,19 @@
+@import url("./card.css");
+@import url("./stars.css");
+@import url("./vars.css");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-color);
+}
```

Comment:
font-size & line-height   - they are almost everywhere different. So you may not indicate them here